Dockerfile

```diff
@@ -57,4 +45,8 @@ RUN pnpm install && \
 EXPOSE 443
 EXPOSE 80
 EXPOSE 9250
+
+HEALTHCHECK --interval=30s --timeout=5s --start-period=10s --retries=3 \
+  CMD curl -f http://localhost:${PROXY_SERVER_HTTP_PORT}/ || exit 1
```

Comment:
The `/status` endpoint is what we should hit to test the server is up.

Reply:
Also, it seems the server url may be a complicated endeavor. We have multiple possible endpoints:

| Config | URL |
|----------|-----|
| HTTP | `http://localhost:${PROXY_SERVER_HTTP_PORT}/status` |
| HTTPS | `https://localhost:${PROXY_SERVER_HTTPS_PORT}/status` |
| Neptune Notebook | `http://localhost:9250/status` |